neqo-http3/tests/httpconn.rs

```diff
@@ -111,6 +112,38 @@ fn connect_peers(hconn_c: &mut Http3Client, hconn_s: &mut Http3Server) -> Option
     out.dgram()
 }
 
+fn connect_peers_with_rtt(
+    hconn_c: &mut Http3Client,
+    hconn_s: &mut Http3Server,
+    net_delay: u64,
+) -> (Option<Datagram>, Instant) {
+    let net_delay = Duration::from_millis(net_delay);
```

Comment:
API quibble: you are using `net_delay`, which is the one way propagation delay rather than RTT, but the function is named "\_with\_rtt".